universal-application-tool-0.0.1/app/views/questiontypes/ApplicantQuestionRendererFactory.java

```diff
@@ -42,4 +48,24 @@ public ApplicantQuestionRenderer getRenderer(ApplicantQuestion question) {
             "Unrecognized question type: " + question.getType());
     }
   }
+
+  private static QuestionDefinitionBuilder sample(QuestionType questionType) {
```

Comment:
(blocking):

1. this should return a built `QuestionDefinition` since the only call using this method is immediately using `build()`.
2. this should be renamed` questionDefinitionSample` to be clear what this is a sample of.

Reply:
Resolved